src/talk/renderer/TitleBar/TitleBar.vue

```diff
@@ -44,12 +44,15 @@ useHotKey('Escape', pushToRoot)
 	<header id="header" class="title-bar">
 		<div class="title-bar__inner">
 			<div v-if="!OS.isMac"
-				class="title-bar__title-wrapper"
+				class="title-bar__title"
 				role="button"
 				tabindex="0"
 				@click="pushToRoot">
-				<span class="title-bar__title">Nextcloud Talk</span>
-				<span v-if="isPreview" class="title-bar__preview-badge">Preview</span>
+				Nextcloud Talk
+			</div>
+
+			<div v-if="channel !== 'stable'" class="title-bar__channel">
+				{{ channel }}
```

Comment:
Let's only do that for non-MacOS and add the channel name to the menu bar of macOS as well